README.md

```diff
@@ -49,14 +47,11 @@ fn main() {
 
     assert_eq!(damerau_levenshtein("ac", "cba"), 2);
 
-    assert!((normalized_damerau_levenshtein("levenshtein", "löwenbräu") - 0.272).abs() <
```

Comment:
I know these lines are a little ugly, but it's because I keep my lines to an 80 character limit. I plan on using `rustfmt` in the future, but even then, I'll keep the 80 character limit.